packages/front-end/services/metrics.tsx

```diff
@@ -266,6 +266,14 @@ export function formatPercentagePoints(value: number) {
   return `${number} pp`;
 }
 
+export function formatBytes(value: number) {
+  return formatByteSizeString(value, true);
+}
+
+export function formatKilobytes(value: number) {
+  return formatByteSizeString(value / 1000, true);
```

Comment:
Should this be `*1000`? Maybe I'm misunderstanding the formats, but if you tell us a column is in KB and the metric value is 23, I'd expect to call the formatByte method with 23000